build/package.json

```diff
@@ -1,8 +1,11 @@
 {
```

Comment:
We would need to strip out the dev-dependencies as well, not worth it just to maintain the version number across folders.